PageScroll not working when the element inside router outlet

I use the new rc1 "router",but I got some error like this:
node_modules/ng2-page-scroll/src/ng2-page-scroll.directive.ts
(30,5): error TS2322: Type 'EventEmitter<{}>' is not assignable to type 'EventEmitter<boolean>'.
  Type '{}' is not assignable to type 'boolean'.

--- a/src/ng2-page-scroll.directive.ts
+++ b/src/ng2-page-scroll.directive.ts
@@ -28,7 +28,7 @@ export class PageScroll implements OnDestroy {
     public pageScrollInterruptible: boolean;
 
     @Output()
-    pageScrollFinish: EventEmitter<boolean> = new EventEmitter();
+    pageScrollFinish: EventEmitter<boolean> = new EventEmitter<boolean>();
 
     private document: Document;
     private body: HTMLBodyElement;

